store/store_test.go

```diff
@@ -221,6 +221,10 @@ func (t *remoteRepoTestSuite) SetUpTest(c *C) {
 	dirs.SetRootDir(c.MkDir())
 	c.Assert(os.MkdirAll(dirs.SnapMountDir, 0755), IsNil)
 
+	oldSnapdDebug := os.Getenv("SNAPD_DEBUG")
```

Comment:
That seems a bit far-fetched, as it'd imply Setenv can't ever be used in Go at all.

Unless there are more details including why they affect Go given the implementation currently in use, let's not take that kind of theory for granted.